javascript/stimulus_reflex.js

```diff
@@ -54,12 +54,34 @@ const createSubscription = controller => {
       received: data => {
         if (!data.cableReady) return
 
+        let reflexOperations = {}
+
+        for (let name in data.operations) {
+          if (data.operations.hasOwnProperty(name)) {
+            for (let i = data.operations[name].length - 1; i >= 0; i--) {
+              if (
+                data.operations[name][i].stimulusReflex ||
+                (data.operations[name][i].detail &&
+                  data.operations[name][i].detail.stimulusReflex)
+              ) {
+                if (!reflexOperations[name]) reflexOperations[name] = []
+                reflexOperations[name].push(data.operations[name][i])
+                data.operations[name].splice(i, 1)
+              }
+            }
+            if (data.operations[name].length == 0)
```

Comment:
We should probably change to a falsey check like 

```js
!data.operations[name].length` 
```

...or use [Strict Equality Comparison](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) 

```js
data.operations[name].length === 0`
```